backend/src/main/kotlin/rocks/didit/sefilm/web/controllers/ShowingController.kt

```diff
@@ -38,10 +39,10 @@ class ShowingController(private val repo: ShowingRepository,
   private val log = LoggerFactory.getLogger(ShowingController::class.java)
 
   @GetMapping(PATH, produces = arrayOf(MediaType.APPLICATION_JSON_UTF8_VALUE))
-  fun findAll() = repo.findByPrivate(false)
+  fun findAll() = repo.findByPrivate(false).map(Showing::toLimitedShowing)
```

Comment:
Ah makes sense. Maybe we could rename it to `Showing::filterSensitiveFields` or `Showing::withoutSensitiveFields` or something similar?